pkg/component/component.go

```diff
@@ -154,7 +154,7 @@ func GetCurrent(client *occlient.Client, applicationName string, projectName str
 
 // PushLocal push local code to the cluster and trigger build there.
 // asFile indicates if it is a binary component or not
-func PushLocal(client *occlient.Client, componentName string, applicationName string, path string, asFile bool, out io.Writer) error {
+func PushLocal(client *occlient.Client, componentName string, applicationName string, path string, asFile bool, out io.Writer, watchedFiles []string) error {
```

Comment:
calling it `watchedFiles`  just looks weird. There is not watching happening here.
How about just `files`? or `filesToPush` or something like that?